app/src/forms/email/assets/confirmation-number-email.html

```diff
@@ -1,118 +1,259 @@
+                        style="width: 100%">
+                        <tbody>
+                          <tr>
+                            <td style="
+                            direction: ltr;
+                            font-size: 0px;
+                            padding: 20px 0;
+                            text-align: center;
+                            vertical-align: top;
+                          ">
+                              <div class="dys-column-per-100 outlook-group-fix" style="
+                              direction: ltr;
+                              display: inline-block;
+                              font-size: 13px;
+                              text-align: left;
+                              vertical-align: top;
+                              width: 100%;
+                            ">
+                                <table border="0" cellpadding="0" cellspacing="0" role="presentation"
+                                  style="vertical-align: top" width="100%">
+                                  <tbody>
+                                    <tr>
+                                      <td align="center" style="
+                                    font-size: 0px;
+                                    padding: 5px 25px;
+                                    word-break: break-word;
+                                  ">
+                                        <a style="text-decoration: none;" href="https://subscribe.developer.gov.bc.ca">
+                                          <div style="
+                                          color: #ffffff;
+                                          font-family: Oxygen, Helvetica neue, sans-serif;
+                                          font-size: 24px;
+                                          font-weight: 700;
+                                          line-height: 37px;
+                                          text-decoration: none;
+                                          display: block;
+                                        ">
+                                            {{ title }}
+                                          </div>
+                                        </a>
+                                      </td>
+                                    </tr>
+                                  </tbody>
+                                </table>
+                              </div>
+                            </td>
+                          </tr>
+                        </tbody>
+                      </table>
+                    </div>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <!-- Progress Modal Start -->
+            <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation"
+              style="background: #f7f7f7; background-color: #f7f7f7; width: 100%">
```

Comment:
Minor aesthetic detail:  you might want a `max-width: 600px` to make the grey box more centered

Reply:
oops nevermind.. i see you have it a few lines down. hmm

Reply:
i guess we could have more padding on the 'View Submission' button.
I havent tried others but this may just happen in outlook desktop. very tricky to get fixed everywhere i believe.

Reply:
Made styling changes to fit the Outlook Desktop format.